etc/docker/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM fpco/stack-build:lts-16.12 as deps-tamarin
+RUN mkdir /opt/build
+WORKDIR /opt/build
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
It looks like you only need this in the `apt-get update` step for the `tamarin-prover` image later, maybe move the `ENV` statement down there. Or better yet, move it into the `RUN` statement where it is actually needed, as to not pollute the environment of downstream images.